Header header
sensor_msgs/Image rgbImage
sensor_msgs/Image depthImage